src/navigation_table.py

```diff
@@ -69,15 +70,17 @@ def _line_to_row(line: str) -> types_.TableRow:
     if match is None:
         raise NavigationTableParseError(f"Invalid table row, {line=!r}")
 
-    level = int(match.group(1))
+    level = int(match.group(1)) if match.group(1) is not None else default_level
     path: types_.TablePath = (match.group(2),)
     navlink_title = match.group(3)
     navlink_link = match.group(4)
 
     return types_.TableRow(
         level=level,
         path=path,
-        navlink=types_.Navlink(title=navlink_title, link=navlink_link or None),
+        navlink=types_.Navlink(
+            title=navlink_title, link=navlink_link or None, hidden=match.group(1) is None
```

Comment:
Do you think is necessary to add a comment making it clear that if a row does not match the pattern will be set as hidden?

Reply:
I have added a comment, note that that a row is hidden if it doesn't have a level